docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -109,16 +115,22 @@ You are free to set ``method_name`` to ``execute`` if you want your Operator to
 Here's a basic example of how a sensor might trigger deferral::
```

Comment:
Let’s also use this chance to make these `.. code-block:: python` instead so these are better syntax-highlighted.

Reply:
Sure. Just updated.